goss-testing/tests/ncn/goss-check-clock-skew.yaml

```diff
@@ -30,7 +30,7 @@ command:
     {{$testlabel}}:
         title: Check clock skew on k8s and storage nodes
         meta:
-            desc: If this test fails, run {{$check_clock_skew}} for more detail on which nodes have clock skew, and then inspect those node(s) to determine why NTP sync may be failing. Start by running 'systemctl status chronyd.service' on those nodes, as well looking for any chrony related messages in /var/log/messages"
+            desc: If this test fails, see https://cray-hpe.github.io/docs-csm/en-12/operations/node_management/configure_ntp_on_ncns/#fix-broken-configs for steps to remediate clock skew."
```

Comment:
Unmatched quotation mark at the end of the line

Reply:
Will fix this with [CASMINST-4529](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4529)